### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Guy-Gustave**
screen 1
this is the navbar and 1st article

### CODE REVIEW

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @Guy-Gustave!👋 
You have done a wonderful job with your project so far! It looks really good! 👏 👏   There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

### Navigation Bar

- **Large Version**

- [ ] The text in the yellow banner should be left-aligned, not centered. Please compare your clone with the original page and fix this.
The original page 👇 
Screen Shot 2020-06-13 at 11 09 49 PM
Your clone 👇 
Screen Shot 2020-06-13 at 11 09 56 PM

- **Medium Version**

- [ ] The TNW logo should appear at the leftmost part of the navigation bar in the medium screen size. Please check my screenshots below and fix this.
Your clone 👇 
Screen Shot 2020-06-13 at 11 13 34 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 13 24 PM

### Top 3 articles section

- **Large version**

- [ ] the gap between the large article on the left and the two smaller articles on the right is larger than it should be. There should also be a gap between the two articles on the right. Please compare your clone with the original page in the screenshots below and fix this. 
The original page 👇 
Screen Shot 2020-06-13 at 11 25 35 PM
Your clone 👇 
Screen Shot 2020-06-13 at 11 25 25 PM

- [ ] Additionally, the date, time, and author name are not positioned correctly on the articles. Please compare your clone with the original page in the screenshots above and fix this. ☝️ 

-  **Medium Version**

- [ ] The TNW logo in the large article on the left should disappear on the medium screen. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-13 at 11 30 02 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 29 49 PM

- **Small Version**

- [ ] The first article is partially hidden by the navigation bar. Please fix this. 
Your clone 👇 
Screen Shot 2020-06-13 at 11 31 48 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 32 09 PM

- [ ] The date, time, author name, and article titles are not properly positioned on the articles on the small screen. Please compare it with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-13 at 11 33 28 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 33 39 PM
### Latest News Section

- **Medium Version**

- [ ] This section should align to 3 news articles per row in the medium screen. Please see the screenshots below and fix this.
Your clone 👇 
Screen Shot 2020-06-13 at 11 45 06 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 45 40 PM

### Latest Funding section
- **Large version**

- [ ] The links (add info, get updates by email, see more) should be in line with the latest funding heading. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-13 at 11 48 57 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 49 19 PM
 

- **Medium Version**

- [ ] This section should align to 3 articles per row in the medium screen size. Please see the screenshots below.
The original page 👇 
Screen Shot 2020-06-13 at 11 51 22 PM
Your clone 👇 
Screen Shot 2020-06-13 at 11 51 41 PM

### Categories (Apps, Gear, Tech, etc) section
- **Large version**

- [ ] The third element of the first row is not aligned properly with the other elements in the row. Please compare your clone with the original page and fix this. 
The original page 👇 
Screen Shot 2020-06-13 at 11 54 16 PM
Your clone 👇 
Screen Shot 2020-06-13 at 11 54 04 PM



- **Medium Version**

- [ ] This section should align to 2 category boxes per row in the medium screen size. Please compare your clone with the original page in the screenshots below. 
The original page 👇 
Screen Shot 2020-06-13 at 11 56 34 PM
Your clone 👇 
Screen Shot 2020-06-13 at 11 57 45 PM

 ### Latest Deals Section
- **Large version**

- [ ] The "see all deals" link should be aligned to the right, not centered. 
Your clone 👇 
Screen Shot 2020-06-13 at 11 59 14 PM
The original page 👇 
Screen Shot 2020-06-13 at 11 59 42 PM



- **Medium Version**

- [ ] This section should align to 3 deals per row in the medium screen size. Please compare your clone with the original page in the screenshots below. 
The original page 👇 
Screen Shot 2020-06-14 at 12 01 13 AM
Your clone 👇 
Screen Shot 2020-06-14 at 12 01 31 AM



- **Small Version**

- [ ] The "Latest Deals" title and the "see all deals link" should not wrap to the next line on the small screen. Additionally, this section is taking up too much space on the small screen. 
Your clone 👇 
Screen Shot 2020-06-14 at 12 03 15 AM

Screen Shot 2020-06-14 at 12 03 19 AM
The original page 👇 
Screen Shot 2020-06-14 at 12 04 52 AM

### Overall

- [ ] I’m not seeing any linter checks in your project. Please ensure stickler is enabled in your repo and all the files were copied correctly. 
Screen Shot 2020-06-13 at 11 02 19 PM

- [ ] It’s better to delete commented code. If you need to review a previous version of your code, use git history. Please check my comments below and remove any instances of inline comments in your code.
Screen Shot 2020-06-13 at 11 06 49 PM

- [ ] There is a horizontal scroll on your page on both the small and medium screens. Please fix this.
Screen Shot 2020-06-13 at 11 14 31 PM

- [ ] There is excess white space on the right side of your clone and a horizontal scroll on the medium screen size. Please fix this.
Screen Shot 2020-06-13 at 11 19 18 PM

- [ ] You should have media queries for 2 breakpoints however I am only seeing for one breakpoint in your CSS file. 
Screen Shot 2020-06-14 at 12 08 15 AM



****
### [OPTIONAL]

- You can use GitHub actions instead of stickler to check for errors. You can find the instructions on how to set it up [here](https://github.com/microverseinc/linters-config/tree/master/html-css)

- You should be working from a feature branch with a descriptive name. This is specified in [this article](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/85215964/) on pathwright about GitFlow. Please try to add a more descriptive name to your feature branch in future projects.
Screen Shot 2020-06-14 at 12 12 06 AM



****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+  padding-left: 1.5%;
+  margin-top: 5rem;
+  display: flex;
+  flex-flow: column;
+}
+
+/* .grid {
+  display: flex;
+  flex-flow: column;
+} */
```
Comment (BrittanyBlake) [inline css/style.css]: Please remove commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+}
+
+.fund-grid {
+  position: relative;
+  margin-top: 15px;
+}
+
+/* .fund-sec-hidden {
+  display: none;
+} */
```
Comment (BrittanyBlake) [inline css/style.css]: Please remove commented code

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+
+.red-img {
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  /* z-index: 1; */
```
Comment (BrittanyBlake) [inline css/style.css]: Please remove commented code

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,706 @@
...
+  font-size: 24px;
+}
+
+.red-img {
+  padding-left: 10px;
+}
+
+/* .red-img{
+
+} */
```
Comment (BrittanyBlake) [inline css/style.css]: Please remove commented code

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,706 @@
...
+    justify-content: space-around;
+    width: 100%;
+    height: 45px;
+    position: sticky;
+    }
+
+    /* .fixed-nav {
+      position: sticky;
+      margin-top: 42px;
+    } */
```
Comment (BrittanyBlake) [inline css/style.css]: Please remove commented code

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Hello @Guy-Gustave 🖐.

### Nice job implementing the changes 👏.

However, there are still some changes required before we can move forward.

- There are some stickler errors that still need fixing. Please pay attention to that.

- The `article-1 vid` class' height should be increased to match the original.

  Right now 👇

  image.

  Original 👇  
  
  image

- The `latest news` section must have some space on either side and there should be four cards in a row.

  Right now 👇

  image

  Original 👇  

  image

- The `lates funding rounds` section should have some space on either side. The `red-img` class is not showing.

   Right now 👇

  image

  Original 👇  

  image

- The `app-gear` section should have some space on either side.

  Right now 👇 
  
  image

  Original 👇  

  image

- The `latest deals` section must have some space on either side and there should be four cards in a row.

  Right now 👇

  image

  Original 👇  

  image

- The `bottom-context` text should be in one line.

  Right now 👇

  image

  Original 👇  

  image

- Your site is not responsive. In medium and small screens all of the elements are overflowing on the x-axis and the fonts in the top 3 articles section are larger than it should be.
  
  image

  image
  
  image

The main objective of this project is to understand media queries in CSS. I see that you have only used one breakpoint, where 2 breakpoints were needed.

image

### Focus on how the styles are changing in the original website for each breakpoint and try to emulate that.

Apply the suggested changes and request another code review.

Happy coding 👨‍💻

**Comment by Guy-Gustave** [comment]
i hope i have checked all suggested. and thank you. now it is seen on mobile and large screen, well displaying

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle:  :stop_sign: 
A great project so far :clap:  You have done such impressive work :clap:  However there are a few issues that have prevented me from approving the project please fix the following issue before resubmitting. :arrow_down: 

- [ ] There are still stickler issues in your project. You have to fix all errors before your project can be approved kindly fix all the errors :arrow_down: 
linter errors

- [ ] There are some responsiveness issues in your project. Remember, You have to make your project extremely responsive. Kindly fix all the responsiveness issues. :arrow_down: 

responsivness issue

- [ ] If you want to hide some items on your mobile view kindly don't just change the color to white,   instead use `display: none` in your CSS :arrow_down: 
medium_view

Also, I noticed there are a few issues you haven't addressed as advised by the previous TSE. Kindly take a step back and address all the issues as advised. It is a requirement before your project can be approved :smiley: 

If you have any question concerning the review kindly reach me on slack using this **@peterrobert**

### Happy coding  :computer:

**Comment by Guy-Gustave** [comment]
i made all suggested correction and remove all stickler errors
article1

**Comment by santiago-dev96** [comment]
# PR under review by @santiago-rodrig (TSE)

**Comment by Guy-Gustave** [comment]
hello rodrig, i hope you are saying the footer. but on my original page is
like this. i may align on one line but the original was like that. thank
you. and the grid that they told me that they must have  spaces i already
changed. thank you

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

Good job so far! But there are still some things to do.

1. Your navigation bar is not being displayed the same way that it is displayed on the original page

**Original - Small Screens**

Screen Shot 2020-06-15 at 9 22 14 AM

**Original - Medium Screens**

Screen Shot 2020-06-15 at 9 23 02 AM

**Yours - Small Screens**

Screen Shot 2020-06-15 at 9 22 22 AM

**Yours - Medium Screens**

Screen Shot 2020-06-15 at 9 23 10 AM

2. Your **Latest News** section is not being displayed as it is displayed on the original page

**Original - Small Screens**

Screen Shot 2020-06-15 at 9 26 32 AM

**Original - Medium Screens**

Screen Shot 2020-06-15 at 9 26 07 AM

**Yours - Small Screens**

latest_small

**Yours - Medium Screens**

Screen Shot 2020-06-15 at 9 25 48 AM

Please submit another CRR after implementing the required changes, and please, take the time to review your project carefully, and also don't forget to review the page that you are trying to clone thoroughly.

You can find me in slack as **srodrig**, happy coding! :wave:

**Comment by Guy-Gustave** [comment]
thank you. let me focus on it i see medium have problem you're right

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello @Guy-Gustave   👋 
You are doing a great job with this project so far! 💪 Great job applying the feedback you have received from myself and the other TSEs. 🎖️ 🌟 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] There are still some stickler issues that need to be resolved. Please fix these.
Screen Shot 2020-06-16 at 1 33 46 AM

- [ ] The Latest Funding should be vertically aligned with the Latest News section above it. This was pointed out by a previous TSE who asked you to add space on either sides. 
The original page 👇 
Screen Shot 2020-06-16 at 1 43 48 AM
Your clone 👇 
Screen Shot 2020-06-16 at 1 45 36 AM

- [ ] In the Apps, Gear, Tech, etc. section, the article images are not properly aligned with the red banner below them. This is an issue with all of the article categories. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-16 at 1 49 00 AM
The original page 👇 
Screen Shot 2020-06-16 at 1 49 50 AM

- [ ] Your clone is still not responsive on the medium and small screens. This was pointed out on all prior reviews. 
Your clone- Medium screen 👇 
Screen Shot 2020-06-16 at 1 54 46 AM
Your clone- Small screen 👇 
Screen Shot 2020-06-16 at 1 55 21 AM

- There is way too much white space when scrolling to the right of your clone on the medium and small screens.

- [ ] Please review all previous TSE comments and ensure that you have done all the required changes. 



 



****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Nice job making the changes 🎉🙌, but we still need to work on some issues.

- The `app-img align` and `latest-bottom` divs must be aligned, as pointed out by the previous TSE.
- The `latest news`, `latest funding`, `apps, gear, tech` and `latest deals` sections must have some space on either side so that they are in the center of the screen, as I had pointed out earlier.

  image

  image

  image

  image

- Your clone is not yet responsive. The elements are shrinking when the screen is resized, whereas, it should adapt to the screen size and change its styles by using media queries.

### Medium screen

image

### Small screen

image

I really appreciate your not-giving-up mentality. In order to move ahead you need to make sure that you understand how media queries work. This is the objective of this project. Let's make sure to take a step back, go through all of the reviews (including this one) and understand what has been asked of you. Do make sure that you also understand how the original site is structured. I can guarantee you, once you understand how styles change when the same is accessed from different screen sizes, life would be so much easier.

### Take your time to understand the media queries. once you are done, make the changes and submit another code review.

Happy coding 👨‍💻

**Comment by Guy-Gustave** [comment]
thank you for the review. i hope i made suggested changes. thank you. can you review again?

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Guy-Gustave  :wave:
-
You have done a good job leading to this point! Kindly note that while it might be prudent to try and keep up with our partners or at times our set goals, a lot of caution should be taken especially when working on projects that demand much of our attention such as these. A positive thing to note is that the project is coming along quite well, given the applied guidelines pointed by our awesome TSEs :tada: And for that, keep it up.

We, however, do have some few changes to carry out before we proceed (see review comment).
##
The [clone](https://raw.githack.com/Guy-Gustave/the-next-web/feature-branch/index.html) shows a good understanding and usage of building with responsive design. 💯 We, however, do need to ensure nothing is messed up. Kindly compare the differences between the clone and the source page, and carry out the necessary suggestions below. 👇  

## Large Screen

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Guy-Gustave/the-next-web/feature-branch/index.html)|
|:---:|:---:|
|source|clone|

### NavBar
- The main header contents in the main navbar(white) and the underneath(yellow), should be aligned to the far left of their respective navbars. Right now, they appear to be centrally aligned which should not be the case. Also, in the main navbar still, please ensure the contents to the right are aligned to the far-right as compared to how they are positioned right now. Kindly fix this.
### Three Articles
- Increase the height and width of the website's logo to match its original source.
-  Reorder the positioning of the date in the third article of the top three articles (move it to the bottom of the article's headline).
- Reposition the date of the main article (move it to the far right of its container element). Currently, it is centrally positioned which is not the case when compared to the source page.
- Increase the font of the header of the main article to match the source page.
### Middle Section
- Reduce the view width size of the `latest news`, `latest funding`,  `apps, gears, tech etc `, and `latest deals` section to match those of the source page's width.

### Footer
- The Footer also appears to be in need of some changes. Compare the differences below 👇 and work on the suggested changes.

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Guy-Gustave/the-next-web/feature-branch/index.html)|
|:---:|:---:|
|footer|footer-clone|

- When we compare the two, there seems to be different from one another. The goal here is to make them strikingly similar. 
- Ensure the social icons are centrally well placed in their respective container elements, and that their padding is evenly distributed (not touching their container's borders). Take care of the colours as well.
- The `EVENTS`, `ABOUT`, `ADVERTISE`, `JOBS`, and `CONTACT` should all be well spaced out. Take care of the spelling mistakes as well.
- The bottom-most footer elements should be aligned in a single line.

## Medium Screen
### Navbar
- Take note of the colour topography of the main navbar(white). It is currently greyish in colour instead of white. Take care of the overflow of the wordings as well during the screen breakpoint transition from large to medium.

### Latest funding

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Guy-Gustave/the-next-web/feature-branch/index.html)|
|:---:|:---:|
|latest-funding-medium|latest-funding-medium-clone|

- The latest funding section only displays three articles on the medium screen display. Kindly, let's find a way of hiding the fourth one in case of our clone page.

### Footer
- Attention is also drawn to the footer of both small and medium screens. Kindly take note of the two and work on the differences below. 👇 

|[Source Page](https://thenextweb.com/) |[Clone page](https://raw.githack.com/Guy-Gustave/the-next-web/feature-branch/index.html)|
|:---:|:---:|
|footer-medium-small|footer-medium-small-clone|

## Small Screen 
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Guy-Gustave/the-next-web/feature-branch/index.html)|
|:---:|:---:|
|small-screen|small-screen-clone|

- Increase the general height of the top three articles to match those of the source page in the small screens.

## Stylesheets 
- Please make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright). We do have a repeated styling rule here: https://github.com/Guy-Gustave/the-next-web/blob/f815a08ad637cf06698c33db9aea8839097b3c88/css/style.css#L340

#### OPTIONAL:
 - Validate your stylesheet using this [tool](https://jigsaw.w3.org/css-validator/validator)

## README
- Improve the screenshots in the README.
- Make the README look more professional by hiding the live link inside of placeholders to look neater.
##
- If you have any questions concerning the changes, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by Guy-Gustave** [comment]
hey Christian i hope i made almost changes, except the footer because it was like mine. when i start it was like now after i changed and now it seems like before because i changed from Saturday. another change is the latest funding, it was 4 items before. i hope the aim oh the project is now. thank you

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,922 @@
...
+}
+
+.nav1,
+.nav2 {
+  display: none;
+}
+
+.tablet {
+  display: none;
+}
```
Comment (andresporras3423) [inline css/style.css]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,922 @@
...
+  background: linear-gradient(180deg, rgba(255, 31, 42, 0.7) 0%, rgba(0, 0, 0, 0.9) 100%), url(../ressources/article-image2.jpg);
+}
+
+.art3 {
+  background: linear-gradient(180deg, rgba(20, 35, 243, 0.7) 0%, rgba(0, 0, 0, 0.9) 100%), url(../ressources/article-image3.jpg);
+}
+
+.alt-img {
+  display: none;
+}
```
Comment (andresporras3423) [inline css/style.css]: Same styles of selector from line 15. In order to minimize code you can group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**@Guy-Gustave** 
Good job with the improvements in the appearance on your page :muscle:  
Your page meets most of the requirements :clap: :clap: :clap:  
 There is just a detail to improve in your CSS before you can go ahead:
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

### OPTIONAL
- [ ] Use header tag for the content on the top of your page. Also, footer tag should be after main tag, not inside.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## It's all coming together now. You have come a long way. Just a little more to go.

- The `latest funding` section needs to be exactly like the original.
### Clone
image

### Original
image

- This image is coming over the navbar in the mobile view 👇
Annotation 2020-06-17 013840

- The top image in `apps, gear, tech,...` section is shrinking when resized 👇
Annotation 2020-06-17 014031

Original 👇
Annotation 2020-06-17 014049

- At width - 420px (and below), the whole clone (except for the navbar) starts shrinking 👇

Annotation 2020-06-17 014521

- In your `style.css`  file, there is an invalid property -> `padding: auto;` at line 581. The invalid style is for the `.vid` class. Fix that.

Make sure to work on these changes before requesting for code review.

Happy coding 👨‍💻

**Comment by Guy-Gustave** [comment]
i finish all suggested. thank you

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,918 @@
...
+  top: 0;
+  background-color: #eee;
+  z-index: 2;
+}
+
+.tablet,
+.nav1,
+.nav2 {
+  display: none;
+}
```
Comment (andresporras3423) [inline css/style.css:20]: Same styles of the selector as line 103. In order to minimize code, you can group selectors with the same style definition.

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,918 @@
...
+  color: red;
+  align-items: center;
+  justify-content: space-between;
+  width: 95%;
+}
+
+.fund-cap,
+.funding-bottom a {
+  text-decoration: none;
+}
```
Comment (andresporras3423) [inline css/style.css:171]: Same styles of the selector as the line 274. In order to minimize code, you can group selectors with the same style definition.

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,918 @@
...
+
+.grid-2 {
+  display: flex;
+  flex-flow: column;
+}
+
+.grid-3 {
+  display: flex;
+  flex-flow: column;
+}
```
Comment (andresporras3423) [inline css/style.css]: Don't repeat yourself. In order to minimize code, you can group selectors with the same style definition.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Guy-Gustave** 
Nice job implementing the requirements of the last review :clap: :clap: :clap:
There are some improvements to do:
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [ ]  Correct the next problems in your page which appears in the small screen size version of your page:



Compare this with the original page:



Correct also this problem which appears in the medium screen size version of your page:



Now, in the footer, the link must appear in the same line in the large screen size version. Look your footer:



And the footer of the original page:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by Guy-Gustave** [comment]
i hope now everything is okay. even the footer i change

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :mechanic: 
**hello** @Guy-Gustave :wave: 
Great job implementing the suggestions from the previous review. :muscle:  :clap:  You have done a great job so far :+1:  However there are still some changes that are required in order for us to approve this project. 

- [ ] Kindly you didn't fix the issue on the footer as advised by the previous TSE. You need to improve on that. :smile:   Remember your site should look like the clone.
footer

- [ ] Kindly also fix the overflow issue. One of your icons is falling off the container during the mobile view
off the container
 **Remember the main reason for this project is to make sure you learn about responsive design** :smile: 

- [ ] Also on your mobile view there is an overflow on the X-axis. You can fix this by referencing the body tag and hiding the over-flow on the x-axis `overflow-x: hidden`

**kindly improve on those and submit your project for another review**

### HAPPY CODING :muscle:

**Comment by Guy-Gustave** [comment]
I don't know how it happens again I had fixed before. but check again it's okay now

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1:  :fireworks: :1st_place_medal: 
**HELLO** :wave: 
The project now looks great to me. You have done such a good job of fixing all the errors. Am sure you have learned a lot

### ON TO THE NEXT PROJECT
### HAPPY CODING

**Pull request merged**